x-pack/plugins/security/server/lib/authorization/actions/saved_object.ts

```diff
@@ -31,25 +20,4 @@ export class SavedObjectActions {
 
     return `saved_object:${type}/${operation}`;
   }
-
-  public allOperations(typeOrTypes: string | string[]): string[] {
```

Comment:
This logic was moved to the `FeaturePrivilegeSavedObjectBuilder`, as we're only using this here, and it makes more sense to be there now that we have the dedicated feature privilege builders.